packs/orion/actions/ncm_config_download.yaml

```diff
@@ -0,0 +1,26 @@
+---
+description: "Download config(s) from Solarwinds NCM Orion module."
+enabled: true
+entry_point: 'ncm_config_download.py'
+name: "ncm_config_download"
+pack: "orion"
+
+runner_type: "python-script"
+
+parameters:
+  node:
+    type: "string"
+    description: "The node to query for state in the monitoring platform."
+    required: true
+  platform:
+    type: "string"
+    description: "The Orion Platform to act on."
+    required: true
+  configs:
+    description: "Which config(s) to download from NCM."
+    type: "array"
+    enum:
+      - ["running"]
```

Comment:
It works everywhere other than the WebUI, so I'll change to a `.` seperated list and an comment to say array's & enums don't play nice.